components/dashboard/HomeProductCard.tsx

```diff
@@ -26,16 +26,19 @@ export const HomeProductCard: React.FC<HomeProductCardProps> = ({
       }}
     >
       <Card
+        p={5}
         overflow="hidden"
         bgColor="backgroundCardHighlight"
+        borderWidth={0.5}
```

Comment:
I don't think this is doing anything?

Reply:
It just adds the `0.5px` to the border width instead of `1px`

<img width="674" alt="image" src="https://github.com/thirdweb-dev/dashboard/assets/136458308/4efb1698-6778-4b71-a140-74e8ed0a2aef">